docs/scope.md

```diff
@@ -0,0 +1,113 @@
+# Table of contents
+- [Table of contents](#table-of-contents)
+- [About this document](#about-this-document)
```

Comment:
I wouldn't put eveything under thing `About this document`. If you want to keep the entry is fine, but I would put the rest of the bullet entries at least in the same level